recipe/meta.yaml

```diff
@@ -26,17 +26,24 @@ requirements:
     - toolchain
     - cmake
     - eigen
-    - numpy x.x
+    - numpy 1.7.*   # [py27]
+    - numpy 1.9.*   # [py35]
```

Comment:
Cool, that'd be nicer. And yeah, I don't especially care about numpy 1.7 support. :)